### PR TITLE
test_: updated profile tests

### PR DESCRIPTION
**Pull request by churik**
Continuation of https://github.com/status-im/status-go/pull/6055 

Based on info from `settings_getSettings `

Updated of these settings do not produce any new signals

Limitations are described [here](https://discord.com/channels/1210237582470807632/1217172931197800468/1304417489198780489)

### CODE REVIEW

**Review by yevh-berdnyk — APPROVED** [review]

**Comment by churik** [comment]
closed https://github.com/status-im/status-go/issues/6070

File: tests-functional/tests/test_wakuext_profile.py [hunk 1]
```diff
@@ -18,14 +18,37 @@ class TestProfile(StatusBackendTestCase):
...
+
+    @pytest.mark.parametrize(
+        "setting_name, default_value, changed_value",
+        [
+            ("currency", "usd", "eth"),
+            ("messages-from-contacts-only", False, True),
+            ("preview-privacy?", False, True),
+            ("default-sync-period", 777600, 259200),
+            ("appearance", 0, 1),
+            ("profile-pictures-show-to", 2, 1),  #obsolate from v1
```
Comment (anastasiyaig) [inline tests-functional/tests/test_wakuext_profile.py]:
```suggestion
            # obsolete
```
:)

**Review by anastasiyaig — APPROVED** [review]

**Review by antdanchenko — APPROVED** [review]

**Comment by igor-sirotin** [comment]
Not sure what's wrong with Codeclimate.

On first run there was an upload error:
```
[2024-11-12T18:07:53.807Z] Uploading coverage report to CodeClimate
[2024-11-12T18:08:00.390Z] Error: response from https://api.codeclimate.com/v1/repos/5a27e6e598ad8b056c002ca2/test_reports/6733997e3c6077463f56a8a7/test_file_reports/batch.
[2024-11-12T18:08:00.390Z] HTTP 500: 
[2024-11-12T18:08:00.390Z] Usage:
[2024-11-12T18:08:00.390Z]   cc-test-reporter after-build [flags]
[2024-11-12T18:08:00.390Z] 
[2024-11-12T18:08:00.390Z] Flags:
[2024-11-12T18:08:00.390Z]   -s, --batch-size int               batch size for source files (default 500)
[2024-11-12T18:08:00.390Z]   -e, --coverage-endpoint string     endpoint to upload coverage information to (default "https://api.codeclimate.com/v1/test_reports")
[2024-11-12T18:08:00.390Z]   -t, --coverage-input-type string   type of input source to use [clover, cobertura, coverage.py, excoveralls, gcov, gocov, jacoco, lcov, lcov-json, simplecov, xccov, dotcover]
[2024-11-12T18:08:00.390Z]       --exit-code int                exit code of the test run
[2024-11-12T18:08:00.390Z]   -r, --id string                    reporter identifier (default "****")
[2024-11-12T18:08:00.390Z]       --insecure                     send coverage insecurely (without HTTPS)
[2024-11-12T18:08:00.390Z]   -p, --prefix string                the root directory where the coverage analysis was performed (default "/home/jenkins/workspace/status-go_prs_tests_PR-6061")
[2024-11-12T18:08:00.390Z] 
[2024-11-12T18:08:00.390Z] Global Flags:
[2024-11-12T18:08:00.390Z]   -d, --debug   run in debug mode
```

But restart says such report already exist:
```
[2024-11-14T03:29:49.811Z] Uploading coverage report to CodeClimate
[2024-11-14T03:29:49.811Z] time="2024-11-14T03:29:48Z" level=warning msg="Conflict when uploading: A test report for commit 0b00a8cd90d986bc184506041eba887453e12735 already exists, skipping upload" 
```

Rebased on latest develop, let's see.

**Comment by igor-sirotin** [comment]
@churik looks like rebasing fixed the issue. I guess it was some CodeClimate glitch 🤷 
Please let me know if it happens again.

image

**Pull request merged**